skyportal/tests/api/test_listings.py

```diff
@@ -0,0 +1,397 @@
+import uuid
+from skyportal.tests import api
+from skyportal.model_util import create_token
+
+
+def test_add_objects_to_list(user, public_candidate, public_candidate2):
+
+    token_id = create_token(
+        ACLs=["Upload data"], user_id=user.id, name=str(uuid.uuid4())
+    )
+
+    status, data = api(
+        'PUT',
+        'listing',
+        data={
+            'user_id': user.id,
+            'obj_id': public_candidate.id,
+            'list_name': 'favorites',
+        },
+        token=token_id,
+    )
+
+    assert status == 200
+    item1 = data["data"]["listing_id"]  # get the list item ID
```

Comment:
```suggestion
    item1 = data["data"]["id"]  # get the list item ID
```
Response data should have `'id'` key here and everywhere else in PUT or POST responses